packages/model-viewer/src/features/controls.ts

```diff
@@ -594,8 +595,8 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
             pointerId: i - 5678,  // help ensure uniqueness
             pointerType: 'touch',
             target: inputElement,
-            clientX: width * position.x,
-            clientY: height * position.y,
+            clientX: width * position.x + rect.x,
+            clientY: height * position.y + rect.y,
```

Comment:
This was an actual bug that our test didn't catch on our old CI, but did catch in this new framework, apparently because the default HTML has some padding now.